scripts/transactions/getWinner.ts

```diff
@@ -0,0 +1,22 @@
+import * as dotenv from 'dotenv';
+import {ethers} from 'hardhat';
+import getBallotContract from '../utils/getBallotContract';
+
+dotenv.config();
+
+async function getwinningProposal(contractAddress: string): Promise<void> {
+  const ballot = getBallotContract(contractAddress);
+  try {
+    const winningProposalVoteCount = await ballot.winningProposal();
+    const winningProposalName = await ballot.winnerName();
```

Comment:
Best to wrap each promise in its own `try/catch`. Helps to trace errors more easily

Reply:
alright